.gitattributes

```diff
@@ -1 +0,0 @@
-* text eol=lf
```

Comment:
What is the reason for removing this?

Reply:
I think my thing did it procedurally. I can add this back.

Reply:
This was filtered by the gitignore, and got dropped when I ran `git rm -r --cached .` to drop the dist